Remote Tickets info (#746)
EuroPython/website: src/content/pages/remote.mdx

---
title: EuroPython 2024 Remote
subtitle: Join EuroPython 2024 remotely!
social_card: social_card_remote.png
---

# EuroPython 2024 Remote

As happy as we are about finally being able to see many friends in **Prague**
this year, we are also aware that many of you will not be able to make it in-person.
Again, we present you EuroPython 2024 remote:

> wherever you are based, let's celebrate Python and our community, rejoice at our diversity and

Reviewer: ```suggestion
Wherever you are based, let's celebrate Python and our community, rejoice at our diversity,
```

learn and have fun in Prague and beyond! In this spirit, we strive to make the remote participation
as interactive as possible.

### How to participate in the conference remotely?

> - To participate in the conference interactively (with guaranteed access to livestreams), a "remote ticket" registration is required. See details and pricing on the [Tickets information](/tickets#remote-tickets) page;

Reviewer: ```suggestion
- To participate in the conference interactively (andguaranteed access to livestreams), a "remote ticket" registration is required. See details and pricing on the [Tickets information](/tickets#remote-tickets) page.
```

Reviewer 2: Oops

Reviewer: The oops was the missing space?

Reviewer 2: > The oops was the missing space?

Yes! It's all good now.

> - All ticket holders will receive a link to join our Discord server. We will send out invites for you to join our server closer to the conference.

Reviewer: ```suggestion
- All ticket holders will receive a link to join our Discord server for the conference.
```

> *Note: in-person ticket holders will be provided with remote event access.*

Reviewer: ```suggestion
*Note: in-person ticket holders will be provided with remote event access as well.*
```

### What is included in the remote ticket?

> Remote participation is between 10-12 July (Wednesday - Friday):

Reviewer: ```suggestion
Remote participation between 10-12 July (Wednesday - Friday):
```

> - Livestreams of Keynotes, Panel discussions, all 6 tracks of talks.

Reviewer: ```suggestion
- Livestreams of Keynotes, Panel discussions, All 6 tracks of talks.
```
or change Panel to panel.

Reviewer 2: Panel to panel makes more sense IMHO

Reviewer: updated suggestion.

- Live participation (text-based) on Q&A session post-talks.
- Text-based discussions with available speakers.
- Online job board and chat with sponsors at the remote booths.
- On-Demand viewing, which will be available hours after the live stream.

> *Note: Tutorials and Workshops on 8-9 July will not be streamed.*

Reviewer: ```suggestion
*Note: Tutorials and Workshops on 8 and 9 July will not be streamed.*
```

### What changes ###

> This year, we will **NOT** provide open and free access to the livestreams of our conference talks (10-12 July). If you would like to watch EuroPython remotely, please consider purchasing a remote ticket.

Reviewer 3: ... purchasing a remote ticket. Purchasing a remote ticket not only gives you access to the conference but also helps support the community!

Reviewer 2: ```suggestion
This year, we will **NOT** provide open and free access to the livestreams of our conference talks (10-12 July).
If you would like to watch EuroPython remotely, please consider purchasing a remote ticket.
Purchasing a remote ticket not only gives you access to the conference but also helps support the community!
```

Reviewer 2: @laisbsc Is this suggestion OK to commit?

> This year, we will **NOT** provide open and free access to the livestreams of our conference talks (19-21 July). If you would like to watch EuroPython remotely, please consider purchasing a remote ticket.

Reviewer 4: ```suggestion
This year, we will **NOT** provide open and free access to the livestreams of our conference talks (10-12 July). If you would like to watch EuroPython remotely, please consider purchasing a remote ticket.
```
